openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py

```diff
@@ -324,6 +326,16 @@ def post_statements_with_unreconciled_lines(env):
             AND bstl.is_reconciled IS DISTINCT FROM TRUE
         """,
     )
+    # New statements with reconciled items
+    openupgrade.logged_query(
+        env.cr,
+        """SELECT bst.id FROM account_bank_statement bst
+        JOIN account_bank_statement_line bstl ON bstl.statement_id = bst.id
+        WHERE bst.state = 'open' AND bstl.is_reconciled IS TRUE
+        """,
+    )
+    stmt_ids = list({x[0] for x in env.cr.fetchall()})
+    env["account.bank.statement"].browse(stmt_ids).button_post()
```

Comment:
why not do this by query?

Reply:
Already explained in the main comment. The remaining moves won't be posted.